doc/conf.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+project = u'PyFingerprint'
+master_doc = 'PyFingerprint'
+copyright = '2019, Bastian Raschke <bastian.raschke@posteo.de>'
+author = 'Bastian Raschke <bastian.raschke@posteo.de>'
+version = '1.5'
```

Comment:
That would be the third version number in project. Please import from module version. Maybe try also to remove the version in the `setup.py`: It would be great to have only one version number in project to raise.

Reply:
Yeah, you are total right!